app/services/agreements/iaa_order_seeder.rb

```diff
@@ -0,0 +1,45 @@
+module Agreements
+  class IaaOrderSeeder < BaseSeeder
+    private
+
+    def record_class
+      IaaOrder
+    end
+
+    def filename
+      'iaa_orders.yml'
+    end
+
+    def primary_attribute_bundle(config)
+      { 'iaa_gtc_id' => config['iaa_gtc'].id, 'order_number' => config['order_number'] }
+    end
+
+    def process_config(iaa_name, config)
+      match = /^(?<gtc_number>[A-Z0-9]+)-(?<order_number>\d+)$/.match(iaa_name)
+      config['iaa_gtc'] = IaaGtc.find_by!(gtc_number: match[:gtc_number])
+      config['iaa_status'] =
+        IaaStatus.find_by!(name: config['iaa_status'])
+
+      @associated_integrations ||= {}
```

Comment:
I wasn't sure how best to hand off the associated integrations - since they need to be created _after_ the initial order is saved (and that happens in the `run` method in the parent class) this seemed like it could work. It might make sense to add a comment here just so it's clear how these are used.